backend/globaleaks/db/__init__.py

```diff
@@ -54,7 +54,7 @@ def init_db(store):
 
     node = models.Node()
     node.wizard_done = GLSettings.skip_wizard
-    node.receipt_salt = generateRandomSalt()
+    node.password_salt = generateRandomSalt()
```

Comment:
@evilaliv3 my suggestion is to keep the `node.receipt_salt` around on this line. 

That N+1 use is different than all N other uses.